packages/cli/src/commands/setup/ui/libraries/tailwindcss.js

```diff
@@ -59,7 +59,13 @@ const tailwindImportsAndNotes = [
 export const handler = async ({ force, install }) => {
   const rwPaths = getPaths()
 
-  const packages = ['postcss', 'postcss-loader', 'tailwindcss', 'autoprefixer']
+  const packages = [
+    'postcss',
+    'postcss-loader',
+    'tailwindcss',
+    'autoprefixer',
+    'prettier-plugin-tailwindcss',
```

Comment:
@Morishiri It's looking good ! Last thing, I think `prettier-plugin-tailwindcss` should be in the package.json on the root, otherwise that will cause error on the output console for Prettier on VS Code.

Reply:
That probably mean 2 different installation process.